<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="10" passed="7" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-05-03T11:54:14Z" name="Suite" finished-at="2022-05-03T11:54:16Z" duration-ms="2675">
    <groups>
    </groups>
    <test started-at="2022-05-03T11:54:14Z" name="GetRepository" finished-at="2022-05-03T11:54:16Z" duration-ms="2223">
      <class name="com.github.testCase.GetRepo">
        <test-method is-config="true" signature="GetAllRepository()[pri:0, instance:com.github.testCase.GetRepo@445b295b]" started-at="2022-05-03T11:54:14Z" name="GetAllRepository" finished-at="2022-05-03T11:54:16Z" duration-ms="2173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetAllRepository -->
        <test-method signature="checkResponseBody()[pri:1, instance:com.github.testCase.GetRepo@445b295b]" started-at="2022-05-03T11:54:16Z" name="checkResponseBody" finished-at="2022-05-03T11:54:16Z" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseBody -->
        <test-method signature="checkResponseCode()[pri:2, instance:com.github.testCase.GetRepo@445b295b]" started-at="2022-05-03T11:54:16Z" name="checkResponseCode" finished-at="2022-05-03T11:54:16Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseCode -->
        <test-method signature="checkStatusLine()[pri:3, instance:com.github.testCase.GetRepo@445b295b]" started-at="2022-05-03T11:54:16Z" name="checkStatusLine" finished-at="2022-05-03T11:54:16Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusLine -->
        <test-method signature="checkContentType()[pri:4, instance:com.github.testCase.GetRepo@445b295b]" started-at="2022-05-03T11:54:16Z" name="checkContentType" finished-at="2022-05-03T11:54:16Z" duration-ms="2" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Content type did not match]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Content type did not match
at org.testng.Assert.fail(Assert.java:96)
at com.github.utils.AssertionUtils.assertForString(AssertionUtils.java:15)
at com.github.testCase.GetRepo.checkContentType(GetRepo.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentType -->
        <test-method signature="checkResponseTime()[pri:5, instance:com.github.testCase.GetRepo@445b295b]" started-at="2022-05-03T11:54:16Z" name="checkResponseTime" finished-at="2022-05-03T11:54:16Z" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseTime -->
      </class> <!-- com.github.testCase.GetRepo -->
    </test> <!-- GetRepository -->
    <test started-at="2022-05-03T11:54:16Z" name="CreateRepository" finished-at="2022-05-03T11:54:16Z" duration-ms="405">
      <class name="com.github.testCase.AddRepository">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.github.testCase.AddRepository@6a03bcb1]" started-at="2022-05-03T11:54:13Z" name="beforeTest" finished-at="2022-05-03T11:54:14Z" duration-ms="639" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:com.github.testCase.AddRepository@6a03bcb1]" started-at="2022-05-03T11:54:16Z" name="beforeClass" finished-at="2022-05-03T11:54:16Z" duration-ms="392" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="checkResponseBody()[pri:1, instance:com.github.testCase.AddRepository@6a03bcb1]" started-at="2022-05-03T11:54:16Z" name="checkResponseBody" finished-at="2022-05-03T11:54:16Z" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseBody -->
        <test-method signature="checkResponseCode()[pri:2, instance:com.github.testCase.AddRepository@6a03bcb1]" started-at="2022-05-03T11:54:16Z" name="checkResponseCode" finished-at="2022-05-03T11:54:16Z" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Response code is not 201. it is 422]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Response code is not 201. it is 422
at org.testng.Assert.fail(Assert.java:96)
at com.github.utils.AssertionUtils.assertForTrue(AssertionUtils.java:34)
at com.github.testCase.AddRepository.checkResponseCode(AddRepository.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseCode -->
        <test-method signature="checkStatusLine()[pri:3, instance:com.github.testCase.AddRepository@6a03bcb1]" started-at="2022-05-03T11:54:16Z" name="checkStatusLine" finished-at="2022-05-03T11:54:16Z" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Status line did not match & it is HTTP/1.1 422 Unprocessable Entity]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Status line did not match & it is HTTP/1.1 422 Unprocessable Entity
at org.testng.Assert.fail(Assert.java:96)
at com.github.utils.AssertionUtils.assertForString(AssertionUtils.java:15)
at com.github.testCase.AddRepository.checkStatusLine(AddRepository.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusLine -->
        <test-method signature="checkContentType()[pri:4, instance:com.github.testCase.AddRepository@6a03bcb1]" started-at="2022-05-03T11:54:16Z" name="checkContentType" finished-at="2022-05-03T11:54:16Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkContentType -->
        <test-method signature="checkResponseTime()[pri:5, instance:com.github.testCase.AddRepository@6a03bcb1]" started-at="2022-05-03T11:54:16Z" name="checkResponseTime" finished-at="2022-05-03T11:54:16Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseTime -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:com.github.testCase.AddRepository@6a03bcb1]" started-at="2022-05-03T11:54:16Z" name="afterTest" finished-at="2022-05-03T11:54:17Z" duration-ms="228" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.github.testCase.AddRepository -->
    </test> <!-- CreateRepository -->
  </suite> <!-- Suite -->
</testng-results>
